WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -95,6 +95,11 @@ final class OrderDetailsViewController: UIViewController {
         super.viewWillAppear(animated)
         syncEverything { [weak self] in
             self?.topLoaderView.isHidden = true
+
+            /// We add the refresh control to the tableview just after the `topLoaderView` disappear for the first time.
+            if self?.tableView.refreshControl == nil {
+                self?.tableView.refreshControl = self?.refreshControl
+            }
```

Comment:
Should we then take the refresh control out in `viewWillDissapear`?
I mean, in order to not allow the loading + pulling when the user goes away and then back to this screen.

Reply:
I think not, because doing so would mean that when the user returns to the screen, we will re-add the control to the view, and he would be allowed to redo pull to refresh while there is already a list of API calls in the background.

Reply:
I'm going to merge it now since it's a hot fix, but we can continue the discussion and eventually if something comes out, I can open a new PR 👍